src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Inputs/BitTextFieldTests.cs

```diff
@@ -0,0 +1,267 @@
+            DataRow(15, true),
+            DataRow(15, false)
+        ]
+        public void BitTextFieldShouldTakeMaxLength(int maxLength, bool isMultiLine)
+        {
+            var component = RenderComponent<BitTextFieldTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.MaxLength, maxLength);
+                    parameters.Add(p => p.IsMultiLine, isMultiLine);
+                });
+
+            var bitTextField = isMultiLine ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
+
+            Assert.IsTrue(bitTextField.HasAttribute("maxlength"));
+            Assert.AreEqual(bitTextField.GetAttribute("maxlength"), maxLength.ToString());
+        }
+
+        [DataTestMethod,
+            DataRow("this is placeholder", true),
+            DataRow("this is placeholder", false),
+        ]
+        public void BitTextFieldShouldTakePlaceHolder(string placeholder, bool isMultiLine)
+        {
+            var component = RenderComponent<BitTextFieldTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.Placeholder, placeholder);
+                    parameters.Add(p => p.IsMultiLine, isMultiLine);
+                });
+
+            var bitTextField = isMultiLine ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
+
+            Assert.IsTrue(bitTextField.HasAttribute("placeholder"));
+
+            Assert.AreEqual(bitTextField.GetAttribute("placeholder"), placeholder);
+        }
+
+        [DataTestMethod,
+            DataRow(true),
+            DataRow(false),
+        ]
+        public void BitTextFieldShouldRespectReadOnly(bool isMultiLine)
+        {
+            var component = RenderComponent<BitTextFieldTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.IsMultiLine, isMultiLine);
+                    parameters.Add(p => p.IsReadOnly, true);
+                });
+
+            var bitTextField = isMultiLine ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
+
+            Assert.IsTrue(bitTextField.HasAttribute("readonly"));
+        }
+
+        [DataTestMethod,
+            DataRow("Emoji2")
+        ]
+        public void BitTextFieldShouldAcceptIcon(string iconName)
+        {
+            var component = RenderComponent<BitTextFieldTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.IconName, iconName);
+                });
+
+            var bitTextFieldIcon = component.Find(".bit-txt span i");
+
+            Assert.IsTrue(bitTextFieldIcon.ClassList.Contains($"bit-icon--{iconName}"));
+        }
+
+        [DataTestMethod,
+            DataRow(true, false),
+            DataRow(true, true),
+            DataRow(false, false),
+            DataRow(false, true),
+        ]
+        public void BitTextFieldCanRevealPassword(bool isEnabled, bool isMultiLine)
```

Comment:
please remove the "isMultiLine" argument from this test. it does not affect a password textfield component.
and also rename the parameter to "IsMultiline"

Reply:
done